### DISCUSSION TITLE
Add 3d-secure-2 support for Stripe

### DISCUSSION THREAD

**Discussion by barchard** [duncanmcclean/simple-commerce#884] [Feature Requests]
I recently lost a dispute due to this issue:

>Finally, your site is not 3-D Secure and does not allow our cardholder to authenticate himself. Conse-
quently, the merchant remains liable for the prejudice

Can we add this to the Stripe integration? 

https://stripe.com/guides/3d-secure-2

Thanks

**Comment by duncanmcclean** [comment]
Simple Commerce already supports 3D Secure 2 by using Stripe's Payment Intents API ([documented by Stripe here](https://stripe.com/docs/payments/payment-intents) and [implemented in Simple Commerce here](https://github.com/duncanmcclean/simple-commerce/blob/5.x/src/Gateways/Builtin/StripeGateway.php#L84)).

You can test the 3DS integration using [Stripe's test cards](https://stripe.com/docs/testing#regulatory-cards) in your local/staging environment.
